src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -83,11 +95,19 @@ export class QuickBar extends LitElement {
   protected renderCommandsList = memoizeOne((filter) => {
     const items = this._filterCommandItems(this._commandItems, filter);
 
+    this._topCommandResult = items[0];
+
     return html`
-      <mwc-list @selected=${this._processItemAndCloseDialog}>
+      <mwc-list activatable @selected=${this._processCommand}>
         ${items.map(
-          (item) => html`
-            <mwc-list-item .item=${item} graphic="icon">
+          (item, index) => html`
+            <mwc-list-item
+              .activated=${index === 0}
```

Comment:
We should drop activated, it is confusing that this does not update when you move through the list with the up/down keys

Reply:
I think it's important to have something showing a highlight while you're typing to indicate that this item will launch if you hit enter. This is how other quick bars do it.

What about _moving_ activated along with the arrow key?

Reply:
Implemented the moving activation highlight. Feels good.

Reply:
Demo: 
![DEMO_movingactivated](https://user-images.githubusercontent.com/1480827/95895693-e4f08200-0d3f-11eb-899e-14a3529a3ce4.gif)